dbcon/joblist/tuplehashjoin.cpp

```diff
@@ -132,8 +132,9 @@ TupleHashJoinStep::~TupleHashJoinStep()
     if (ownsOutputDL)
         delete outputDL;
 
-    for (uint i = 0 ; i < smallDLs.size(); i++)
-        resourceManager->returnMemory(memUsedByEachJoin[i], sessionMemLimit);
+    if (memUsedByEachJoin)
```

Comment:
Not sure what our coding standard is regarding this, but I would definitely add braces around this if statement considering it contains a for loop inside it and not a single statement. Other than that i have no issue with change.

Reply:
I think you're right, our coding standard says always add brackets.  I added them in the join abort fix PR, so will not add them to this one.